RockWeb/Themes/NewSpring/assets/Lava/LAYOUT/editorial-detail.lava

```diff
@@ -7,11 +7,21 @@
 {% assign publishDateTime = Item.PublishDateTime | Date:'yyyyMMddHHmm' %}
 {% assign expireDateTime = Item.ExpireDateTime | Date:'yyyyMMddHHmm' %}
 
-{% if publishDateTime > now or expireDateTime and expireDateTime != empty and expireDateTime <= now or Item == null %}
-    {% if CurrentPersonCanEdit %}
-        <p class="alert alert-danger">If you could not edit you would be redirected to <a href="/page-not-found">/page-not-found</a> as this entry is not active.</p>
-    {% else %}
-        {{ '/page-not-found' | PageRedirect }}
+{% if Item.ChannelId == 17 %}
+    {% if expireDateTime and expireDateTime != empty and expireDateTime <= now %}
+        {% if CurrentPersonCanEdit %}
+            <p class="alert alert-danger">If you could not edit you would be redirected to <a href="/page-not-found">/page-not-found</a> as this entry is not active.</p>
+        {% else %}
+            {{ '/page-not-found' | PageRedirect }}
+        {% endif %}
+    {% endif %}
+{% else %}
+    {% if publishDateTime > now or expireDateTime and expireDateTime != empty and expireDateTime <= now %}
+        {% if CurrentPersonCanEdit %}
+            <p class="alert alert-danger">If you could not edit you would be redirected to <a href="/page-not-found">/page-not-found</a> as this entry is not active.</p>
+        {% else %}
+            {{ '/page-not-found' | PageRedirect }}
+        {% endif %}
```

Comment:
@briankalwat I think this works. Which is cool!

Question: I kind of feel like we could pull out the `expireDateTime` stuff to its own thing? Is this a bad idea?

```suggestion
{% if expireDateTime and expireDateTime != empty and expireDateTime <= now %}
    {% if CurrentPersonCanEdit %}
        <p class="alert alert-danger">If you could not edit you would be redirected to <a href="/page-not-found">/page-not-found</a> as this entry is not active.</p>
    {% else %}
        {{ '/page-not-found' | PageRedirect }}
    {% endif %}
{% endif %}
{% if Item.ChannelId != 17 %}
    {% if publishDateTime > now}
        {% if CurrentPersonCanEdit %}
            <p class="alert alert-danger">If you could not edit you would be redirected to <a href="/page-not-found">/page-not-found</a> as this entry is not active.</p>
        {% else %}
            {{ '/page-not-found' | PageRedirect }}
        {% endif %}
    {% endif %}
{% endif %}
```

Also, do we need the `Item == null` check?

Reply:
Yes, it could be, and that's what I think I tried before with just adding `Item.ChannelId != 17` to the other side of the `and` which, for whatever reason, did not work. This is more complex, but it removes the complex conditionals which aren't super clear as to how they work.